bin/dev

```diff
@@ -1,63 +1,46 @@
 #!/bin/bash
 
-DEV_CONTAINER_NAME="syncm8_dev"
-
-# Find container id of syncm8_dev in order to exec things against it
-# args: name of variable to set container id
-findBackendContainerId(){
-    local  _resultvar=$1
-    local container=$(docker ps|grep $DEV_CONTAINER_NAME|cut -d ' ' -f1)
-    if [ ! $container ]; then
-        bin/util error "No container for dev found. Check that it is running with bin/dev ps, or start it with bin/dev up"
-        exit 1
-    fi
-    eval $_resultvar="'$container'"
-}
+DEVBOX_VERSION=$(cat DEVBOX_VERSION)
 
-# Run commands in exiting dev container as correct user
-# args: commands to run in container
-runDockerExec(){
-    findBackendContainerId container_id
-    cmd="docker exec -it -u $(id -u):$(id -g) -w /home/worker/app/server $container_id "$@
-    bin/util logAndRun $cmd
-}
+DOCKER_COMPOSE="export DEVBOX_VERSION=$DEVBOX_VERSION && docker-compose"
```

Comment:
Can't perform arbitrary shell commands in yaml, but can read the env variables, so explicitly set it before running `docker-compose`.